RIE/dasi/RIE7808dasi.xml

```diff
@@ -137,8 +136,49 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                <div type="edition" source="#rie1">
                 <ab>
                  <!-- text keyed in with Keyman and Diacritics configuration-->
-           <lb n="1" style="text-direction:l-to-r"/><milestone unit="face" n="A"/><supplied reason="lost"><g type="crux"/><g type="disc"/><milestone unit="word"/> lmn
```

Comment:
we should be using `g/@ref` not `g/@type` I wonder what do we have in the GL about this?